### DISCUSSION TITLE
User message : Shared library not found or inaccessible.

### DISCUSSION THREAD

**Discussion by nico-robert** [apnadkarni/tcl-cffi#182] [General]
Hello @apnadkarni ,

It is not a problem with `tcl-cffi`, but with the message when I can't load my library :
`Shared library "myDLL.dll" not found or inaccessible. Could not load shared library.`
My dll is **accessible** and **present** but some of my library dependencies are **missing**.
A better message will be (I think): `Your library is accessible but some of the dependencies are missing.`
Related to my [previous](https://github.com/apnadkarni/tcl-cffi/discussions/114)  question, I don't know if it is possible to know side `tcl-cffi`.

Thanks
Nicolas

**Comment by apnadkarni** [comment]
Moved to Issues.
